tests/bluetooth/bsim_bt/bsim_test_eatt/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ project(bsim_test_host)
 
 target_sources(app PRIVATE
   src/main.c
-  src/main_collision.c
+  src/main_connect.c
```

Comment:
I'm a little confused here. It seems that you are replacing the main_collision test with main_connect. Is that intentional? If so, what about the prj_collision.conf?

Reply:
Just renaming it because the test is not just used for collision anymore.

Reply:
I see, and then you use the `.conf` files to compile different binaries and use that to test different things? 

To me it would seem cleaner to split the test into two, and have a collision test and a auto-connect test.

Reply:
Yes, the same test procedure is used for multiple tests with different configs. The auto-connect one has a slightly different procedure as it does not call `bt_eatt_connect()`. Can move that one out to a separate file.

Reply:
Created two files instead and moved common code to a separate file.